src/main/scala/com/databricks/labs/mosaic/functions/MosaicContext.scala

```diff
@@ -511,6 +511,8 @@ class MosaicContext(indexSystem: IndexSystem, geometryAPI: GeometryAPI) extends
         def st_scale(geom1: Column, xd: Column, yd: Column): Column =
             ColumnAdapter(ST_Scale(geom1.expr, xd.expr, yd.expr, geometryAPI.name))
         def st_setsrid(geom: Column, srid: Column): Column = ColumnAdapter(ST_SetSRID(geom.expr, srid.expr, geometryAPI.name))
+        def st_simplify(geom: Column, tolerance: Column): Column =
```

Comment:
This is now not covered by unit tests.
Could you please cover this one as well?

Reply:
Of course!